src/overlays/actors/ovl_En_Tg/z_en_tg.h

```diff
@@ -2,15 +2,33 @@
 #define Z_EN_TG_H
 
 #include "global.h"
+#include "objects/object_mu/object_mu.h"
 
 struct EnTg;
 
 typedef void (*EnTgActionFunc)(struct EnTg*, PlayState*);
 
+typedef struct EnTgHeartEffect {
+    /* 0x00 */ u8 isEnabled;
+    /* 0x01 */ u8 timer;
+    /* 0x04 */ f32 scale;
+    /* 0x08 */ char unk8[0xC];
+    /* 0x14 */ Vec3f pos;
+    /* 0x20 */ Vec3f unusedZeroVec;
+    /* 0x2C */ Vec3f velocity;
+    /* 0x38 */ s16 angle;
+} EnTgHeartEffect; // size = 0x3C
+
 typedef struct EnTg {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ EnTgActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x400];
+    /* 0x0000 */ Actor actor;
+    /* 0x0144 */ EnTgActionFunc actionFunc;
+    /* 0x0148 */ SkelAnime skelAnime;
```

Comment:
You seemed to have added extra leading 0s, could you fix that?
`0x0144` -> `0x144`